src/movie-card.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './movie-card.css';
+
+const Card =({id, img, title}) => {
+  return (
+    <div className='card' key={id}>
+      <img className='poster' src={img} alt={`Poster of ${title} movie`}/>
+      <h3>{title}</h3>
+    </div>
+  )
+}
+
+export default Card;
```

Comment:
Really like how the alt text was made dynamic reflecting the specific movie poster.